### DISCUSSION TITLE
Get environment from different files under a folder rather than just one data.json

### DISCUSSION THREAD

**Discussion by akashvekariya** [mockoon/mockoon#1087] [Help]
Our requirement is that we have so many mock API's and they change every now and then, we've setup pipeline to push it to mock production environment. But, we're currently using wiremock where I can just give the path of the folder and it will take every api configuration from seperate files from that folder. Here's example

![Screenshot 2023-07-19 at 11 54 14 AM](https://github.com/mockoon/mockoon/assets/59219441/832e7ee7-70f8-4ba8-85e7-66b72b5e52f1)

![Screenshot 2023-07-19 at 11 52 58 AM](https://github.com/mockoon/mockoon/assets/59219441/9cf8b4e7-0ba7-427a-ba92-643a928afe6a)

Although this is not the official wiremock-ui, it's third party repo. We want something like this in mockoon as mockoon supports randomness and templating in response. How do I achieve this kind of configuration where I can pass folder rather than just file name

From this `mockoon-cli start --data ~/data.json` to `mockoon-cli start --data-folder ~/data/`

**Comment by 255kb** [comment]
This is not possible yet. There is an open issue that you can upvote [there](https://github.com/mockoon/mockoon/issues/877)
